addons/mail/static/src/core/common/message.js

```diff
@@ -511,4 +517,32 @@ export class Message extends Component {
             message: this.props.message,
         });
     }
+
+    // TODO KFR: Create something like a translation data object
+    async attemptTranslation() {
+        const response = await this.rpc("/discuss/translation/translate", {body: this.message.body})
+        if (response?.value) {
+            this.message.translatedBody = markup(response.value);
+            this.message.translationSource = response.source;
+        }
+        else {
+            this.message.translationErrors = true;
+        }
```

Comment:
In discuss code, we have `web/tooling` enabled, so that all our code uses prettier/eslint with same configs across all JS modules that also have them. Many teams do have, so it's pretty cool to barely care for code styling when it's automated and standardized :) I strongly recommend you take the time to configure it with your IDE. You can look at `web/tooling/enabled.sh` for more info.

Reply:
Awesome I enabled the tooling